Documentation/CLionConfiguration.md

```diff
@@ -16,7 +16,7 @@ After opening the `serenity` repository in CLion as a new project, the "`Open Pr
 
 > _If you have not built the Toolchain at this point, please do so: `./Toolchain/BuildIt.sh`_
 
-If you already have the project open, you can go to "`File->Settings->Build, Execution, Deployment->Cmake`" to find these options.
+If you already have the project open, you can go to `File -> Settings -> Build, Execution, Deployment -> Cmake` to find these options.
```

Comment:
Since you're touching this line, could also fix Cmake -> CMake :^)

Reply:
*cannot unsee*